imports/client/components/ProfileList.tsx

```diff
@@ -367,6 +384,79 @@ const ProfileList = ({
     );
   }, [hunt, canSyncDiscord, syncDiscord]);
 
+  const invitationLink = useMemo(() => {
+    if (!hunt || !canInvite || !invitationCode) {
+      return null;
+    }
+
+    const copyTooltip = <Tooltip>Copy to clipboard</Tooltip>;
+
+    const invitationUrl = `${window.location.origin}/join/${invitationCode}`;
+
+    return (
+      <p>
+        Invitation link:{" "}
+        <OverlayTrigger placement="top" overlay={copyTooltip}>
+          {({ ref, ...triggerHandler }) => (
+            <CopyToClipboard text={invitationUrl} {...triggerHandler}>
+              <StyledLinkButton ref={ref} variant="link" aria-label="Copy">
+                <FontAwesomeIcon icon={faCopy} fixedWidth />
+              </StyledLinkButton>
+            </CopyToClipboard>
+          )}
+        </OverlayTrigger>{" "}
+        {invitationUrl}
+      </p>
+    );
+  }, [hunt, canInvite, invitationCode]);
+
+  const generateInvitationLink = useCallback(() => {
+    if (!hunt) {
+      return;
+    }
+
+    generateHuntInvitationCode.call({ huntId: hunt._id });
+  }, [hunt]);
+
+  const clearInvitationLink = useCallback(() => {
+    if (!hunt) {
+      return;
+    }
+
+    clearHuntInvitationCode.call({ huntId: hunt._id });
+  }, [hunt]);
+
+  const invitationLinkManagementButtons = useMemo(() => {
+    if (!hunt || !canUpdateHuntInvitationCode) {
+      return null;
+    }
+
+    return (
+      <FormGroup className="mb-3">
+        <Button variant="info" onClick={generateInvitationLink}>
+          {invitationCode
+            ? "Regenerate invitation link"
+            : "Generate invitation link"}
+        </Button>
+        {invitationCode && (
+          <Button variant="info" className="ms-1" onClick={clearInvitationLink}>
+            Disable invitation link
+          </Button>
+        )}
+        <FormText>
+          Manage the public invitation link that can be used by anyone to join
+          this hunt
+        </FormText>
+      </FormGroup>
+    );
+  }, [
+    hunt,
+    canUpdateHuntInvitationCode,
+    clearInvitationLink,
+    generateInvitationLink,
+    invitationCode,
+  ]);
```

Comment:
I'm going to channel @zarvox for a sec and comment that there's no user feedback when you click on these buttons. At minimum, they should probably be disabled while the API call is in flight. Drew would probably ask that errors get presented if there is one, although I'm generally a little less picky about that. (I think in this case we probably don't need an explicit indicator of success, since that's reflected in the button/link content visually changing)

Reply:
Makes sense. I went ahead and added confirmation dialogs for all of the operations, repurposing the existing dialogs for promoting/demoting operators. Those dialogs already handle in-flight button disabling and showing errors, so this should now be covered.